modules/microshift-install-bootc-pb-copy-list.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-bootc-physically-bound.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-install-bootc-pb-copy-list_{context}"]
+= Copying and listing images for building
```

Comment:
I would change the title and high-level description to reflect the essense of what we're trying to do, i.e. embed container images into a bootc image we're building.